packages/client/components/SpotlightResults.tsx

```diff
@@ -88,33 +87,34 @@ const SpotlightGroups = (props: Props) => {
   )
   const {viewer} = data
   const {meeting, similarReflectionGroups} = viewer
-  const {spotlightSearchQuery} = meeting!
-
   const resultsRef = useRef<HTMLDivElement>(null)
   const groupMatrix = useGroupMatrix(similarReflectionGroups, resultsRef, phaseRef)
-  const scrollHeight = useResultsHeight(resultsRef, spotlightSearchQuery || '')
+  const scrollHeight = useResultsHeight(resultsRef)
 
-  if (!similarReflectionGroups.length) return <SpotlightGroupsEmptyState />
   return (
-    <SimilarGroups>
-      <Scrollbar height={scrollHeight} ref={resultsRef}>
-        {groupMatrix.map((row) => (
-          <Column key={`row-${row[0].id}`}>
-            {row.map((group) => (
-              <ReflectionGroup
-                key={group.id}
-                meetingRef={meeting!}
-                phaseRef={phaseRef}
-                reflectionGroupRef={group}
-                expandedReflectionGroupPortalParentId='spotlight'
-                isSpotlightEntering={isSpotlightEntering}
-              />
-            ))}
-          </Column>
-        ))}
-      </Scrollbar>
-    </SimilarGroups>
+    <ResultsWrapper>
+      {!similarReflectionGroups.length ? (
+        <SpotlightResultsEmptyState height={scrollHeight} />
```

Comment:
EmptyState needs to inherit the padding in `ResultsWrapper`. There's a few different ways we could do this, and I don't love the ternary operator within the JSX, but I prefer this DRY approach vs copying padding to empty state component, two result statements wrapped in the same `ResultsWrapper`, or adding it to the parent component that doesn't contain any styles.